### DISCUSSION TITLE
Notifications wikipad

### DISCUSSION THREAD

**Discussion by ghost** [WikidPad/WikidPad#86] [Q&A] [answered]
Hi everyone. 

Would it be possible to be notified of updated documentation on wikipad? 

*For example, on GitHub there are notifications about updates to repositories, comments, discussions - would it be possible to have this same feature on Wikipad?*

**Comment by BjornFJohansson** [comment, marked as answer]
Not afaik, but you are welcome to create such functionality if you need it. I keep my wiki files in a dropbox that does this for me indirectly.

**Reply by ghost** [reply]
Hi BjornFJohansson.

Wikipad is made in python, there is [notify-py](https://pypi.org/project/notify-py/) library(made in python) which can notify updates etc. wdyt(what do you think) ?

#### example.py 
```python 
from notifypy import Notify

notification = Notify()
notification.title = "Update"
notification.message = "Wikipad community, new update"
notification.send()
```
